### ISSUE TITLE
I cannot open downloaded mangas while offline

### ISSUE THREAD

**Issue by RodsonMatos** [KotatsuApp/Kotatsu#1142]
### Brief summary

When offline tha app starts trying to load the ui as if it was online and none of my dowloaded mangas can't open, from any source or tab.

### Kotatsu version

7.6.3

### Android version

12

### Device

Motorola Moto e22i

### Acknowledgements

- [X] This is not a duplicate of an existing issue. Please look through the list of [open issues](https://github.com/KotatsuApp/Kotatsu/issues) before creating a new one.
- [X] This is not an issue with a specific manga source. Otherwise, you have to open an issue in the [parsers repository](https://github.com/KotatsuApp/kotatsu-parsers/issues/new/choose).

**Comment by MariusAlbrecht** [comment]
can't reproduce the issue as described.
There's probably missing information.

**Comment by RodsonMatos** [comment]
I don't know if it's reproduceable.

**Comment by Sensi116** [comment]
Me to brother 😞

**Comment by pyche2** [comment]
I can reproduce, haven't been able to read mangas offline for more than a year but can't find a way to get logs or something useful to communicate.

I have the F-droid version 7.6.4  
 Android 12 /e/OS  
 Device Zenfone 8

**Comment by MariusAlbrecht** [comment]
so you can open them, it just takes an unreasonable amount of time?
Is that the case for everyone?

**Comment by pyche2** [comment]
Yes that's it, it took me 1min58sec to open it. I've launched the app through android studio on my phone and those are the logs in logcat : [logKotatsu.txt](https://github.com/user-attachments/files/17535339/logKotatsu.txt)

**Comment by RodsonMatos** [comment]
I guess it's solved now.

**Comment by Sensi116** [comment]
It takes me more than 8 minutes ⁦(⁠T⁠T⁠)⁩

في الأحد، 27 أكتوبر 2024 7:38 م Marius Albrecht ***@***.***>
كتب:

**Comment by RodsonMatos** [comment]
> It takes me more than 8 minutes ⁦(⁠T⁠T⁠)⁩
Do you have by any chance a VPN enabled?

**Comment by Sensi116** [comment]
Yes baby 😋

في الأحد، 5 يناير 2025 5:14 م Rodson Matos ***@***.***> كتب:

**Comment by Sensi116** [comment]
No it has been solved thank you

في الأحد، 5 يناير 2025 5:15 م SINSI ***@***.***> كتب:
